test/e2e/support/utils.js

```diff
@@ -272,6 +290,17 @@ export async function clickOnMoreMenuItem( buttonLabel ) {
 	await itemButton.click( 'button' );
 }
 
+/**
+ * Clicks on Block Settings Menu item, searches for the button with the text provided and clicks it.
+ *
+ * @param {string} buttonLabel The label to search the button for.
+ */
+export async function clickOnBlockSettingsMenuItem( buttonLabel ) {
+	await expect( page ).toClick( '.editor-block-settings-menu__toggle' );
+	const itemButton = ( await page.$x( `//button[contains(text(), '${ buttonLabel }')]` ) )[ 0 ];
+	await itemButton.click( 'button' );
```

Comment:
Is the argument necessary? Seems `click()` would occur on the button anyways.

Reply:
Now removed.